config/localization/en/app.json

```diff
@@ -989,7 +989,8 @@
       "AMOUNT_TO_STAKE": "Amount to Stake",
       "SELECTED_VALIDATOR": "Selected Validator",
       "ENTER_AMOUNT_TO_STAKE": "Enter amount to stake",
-      "STAKED_BALANCE": "Staked Balance"
+      "STAKED_BALANCE": "Staked Balance",
```

Comment:
i feel like staked balance got deleted at some point because the key exists in the generated file, but not here 🤷‍♀️